giza_actions/integrations/uniswap/addresses.py

```diff
@@ -0,0 +1,39 @@
+ADDRESSES = {
+    # mainnet
+    1: {
+        3: {
```

Comment:
Why there is always this `3` as a key in all the addresses?

Reply:
uniswap version 3. I don't have v2 support yet but it definitely should be one of the next todos